.gitignore

```diff
@@ -32,7 +32,6 @@ generated_testSrc/
 out/
 atlasdb-cli/var/data/
 *.hprof
-var/
```

Comment:
Intentional? We don't seem to be adding any files to the `var` folder in this PR

Reply:
yup, this include the the timelock-server-distribution/var/conf files, which was added by this PR.

Reply:
could we rename that folder to "resource" instead? I feel we have several `var` folders:
```
./atlasdb-api/var
./atlasdb-cassandra-multinode-tests/var
./atlasdb-cli-distribution/build/gjd-tmp/var
./atlasdb-console-distribution/build/gjd-tmp/var
./atlasdb-ete-tests/build/gjd-tmp/var
./atlasdb-jdbc-tests/var
./atlasdb-perf/var
./atlasdb-service-server/src/dist/var
./timelock-server/build/gjd-tmp/var
./timelock-server/var
./timelock-server-distribution/build/gjd-tmp/var
./timelock-server-distribution/var
```
that are probably populated on tests / builds. This would also make follow our pattern of having yml files in resources folders.

Reply:
No, var/conf is the default conf path for sls spec and a lot of test setups have this hardcoded.. renaming seems a lot of overhead, we can do some gitignore magic to unignore `./timelock-server-distribution/var` specifically if you would like.. but its time-consuming and I am not sure if its worth the effort.

Reply:
Sure. So if we want to keep the folder named this way, we can readd the `var/` on .gitignore. The protections of .gitignore are just applied after the .gitignore is commited, this is:

Let's say you have a `a/` on .gitignore and you want to add a file `a/b`. You can:
- remove the `a/` from .gitignore on commit 1;
- add the file `a/b` on commit 2;
- re-add the `a/` on .gitignore on commit 3.

This will make us preserve the previous behavior while having the new file. This was also probably how the other files on that folder were added to git.

Reply:
Done